src/store/auth.module.js

```diff
@@ -0,0 +1,61 @@
+import AuthService from '../services/auth.service';
```

Comment:
dont use relative paths for imports
```suggestion
import AuthService from '@/services/http/auth.service';
```

Reply:
Cannot install Node.js module: "C:\Program Files\nodejs\node.exe" "C:\Program Files\nodejs\node_modules\npm\bin\npm-cli.js" install @/services --save-dev

Reply:
It's not a module to install. It's a path relative to src directory: https://stackoverflow.com/a/42753045/10876483